telco-core/configuration/reference-crs-kube-compare/required/networking/metallb/addr-pool.yaml

```diff
@@ -13,5 +13,5 @@ spec:
 {{ .spec.addresses | toYaml | indent 2}}
 {{ end }}
   #- 3.3.3.0/24
-  autoAssign: true
+  autoAssign: false
```

Comment:
is this really what we want?

Reply:
the justification is that if _IPAddressPool_ contains multiple IP addresses then static assignments will ensure that _CNF-A_ always has IP address _X.X.X.X_(unless IPAddressPool is solely dedicated to the _CNF-A_)

Reply:
if that's a requirement, the right way to address it is to use the service / namespace selectors in the ip address pools (assuming each CNF will have its own namespace, but that can be done per service too).

This guarantees each cnf will have the ip the admin means to assign to them, otherwise they can steal the ip from any pool.

Reply:
good points @fedepaol. And while without  _service/namespace_ selectors IP address from any IPAddressPool could be selected, imho `autoAssign` set to _false_ is a good safety net from assigning _not expected_ IP addresses.

Reply:
@fedepaol , I added _serviceAllocation_ stanza to the CR, wdyt?